app/src/main/res/layout/activity_login.xml

```diff
@@ -53,7 +53,7 @@
                         android:paddingTop="@dimen/small_gap"
                         android:textAlignment="center"
                         android:textColor="@color/secondaryDarkColor"
-                        tools:text="Use Commons Beta Credentials" />
+                        tools:text="You are using Commons Beta Credentials" />
 
```

Comment:
Can you please call this text also from strings file?

Reply:
Will do !